package.json

```diff
@@ -16,7 +16,8 @@
     "registry": "https://registry.npmjs.org/"
   },
   "scripts": {
-    "build": "gulp && npm run build:stylesheets",
+    "build": "gulp && npm run build:sass && npm run build:stylesheets",
+    "build:sass": "backpack-node-sass --prefixComment=\"$(cat ./license.txt)\" && rm packages/bpk-stylesheets/index.css",
```

Comment:
I'm a bit confused, what is this cmd supposed to do and why are we removing the stylesheets index.css?

Reply:
So this command, which we changed from just the transpiled process to regular build is: 

- To compile the Sass files to CSS but does it in a parallel manner for speed vs one file by one file.
- For legal copyright appends the licence header to each file

For the removal of the file, we don't want this file transpiled because it will be unusable and don't want systems to attempt use as it generates links to local machine directories for Normalise CSS.

This command was updated based on what we used to have in Backpack before we moved these to foundations - see here for reference: https://github.com/Skyscanner/backpack/pull/2381/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L21

Reply:
For reference this was where it was before in the transpilation process: https://github.com/Skyscanner/backpack/pull/2959/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L57